### DISCUSSION TITLE
Let `vue-tsc` produce a warning/error for unknown/untyped elements

### DISCUSSION THREAD

**Discussion by AlbertBrand** [vuejs/language-tools#276] [Ideas]
It seems unknown elements are skipped entirely when performing the typecheck. I would actually love to see a warning or error for those cases, because it often means:
- someone accidentally broke an import
- someone forgot to register the component type globally

I think the Angular template typecheck is pretty strict in those cases and it would be a good addition for strictness.

**Comment by johnsoncodehk** [comment]
This requirement still needs some discussion, because some users may not expect this behavior.

If it should implement, I may add a new option as a switch in tsconfig.json.
